source/whats-new.txt

```diff
@@ -42,6 +43,143 @@ Learn what's new in:
+
+{+libbson+}
+~~~~~~~
+
+The 2.0.0 release of ``{+libbson+}`` includes the following changes:
+
+New Features
+````````````
+
+- Adds support for the BSON Binary Vector subtype. To learn more, see
+  `BSON Binary Vector subtype <{+api-libbson+}/binary_vector.html>`__ in
+  the {+libbson+} API documentation.
+
+Changes
+```````
+
+- Reduces the value of the ``BSON_ERROR_BUFFER_SIZE`` macro from ``504``
+  to ``503`` to reserve the final byte for internal use.
+
+- Renames the following ``pkg-config`` files:
+
+  - ``libbson-1.0``: Renamed to ``bson2``
+  - ``libbson-static-1.0``: Renamed to ``bson2-static``
+
+- Removes the ``bson-1.0`` CMake package. Instead, use ``bson`` and
+  specify a version or version range to be imported, as shown in the
+  following example:
+
+  .. code-block:: c
+     
+     find_package(bson 1.30...2.0)
+
+- Renames the following imported targets:
+
+  - ``mongo::bson_static``: Renamed to ``bson::static``
+  - ``mongo::bson_shared``: Renamed to ``bson::shared``
+
+- Adds the ``bson::bson`` target, which points to either the static
+  library or shared library depending on the value of the
+  ``BSON_DEFAULT_IMPORTED_LIBRARY_TYPE`` CMake option. If you don't set this option and
+  both library types are available, ``bson::bson`` uses
+  the static library.
+
+{+libmongoc+}
+~~~~~~~~~
+
+The 2.0.0 release of ``{+libmongoc+}`` includes the following changes:
+
+New Features
+````````````
+
+- Supports aggregate operations that include ``$lookup`` stages for
+  In-Use Encryption. To learn more, see the following
+  references in the {+mdb-server+} manual:
+
+  - :upcoming:`$lookup and $graphLookup Behavior </core/csfle/reference/supported-operations/#-lookup-and--graphlookup-behavior>` for CSFLE
+  - :upcoming:`$lookup and $graphLookup Behavior </core/queryable-encryption/reference/supported-operations/#-lookup-and--graphlookup-behavior>` for QE
+
+Changes
+```````
+
+- If you pass the ``batchSize:0`` option to ``mongoc_client_watch``, ``mongoc_database_watch``,
+  or ``mongoc_collection_watch``, the option also applies to the ``aggregate`` command.
+
+- Removes ``bson_oid_init_sequence``. Instead, use ``bson_oid_init``.
+
+- Changes the return type of ``mongoc_server_description_host`` from ``mongoc_host_list_t *``
+  to ``const mongoc_host_list_t *``.
+
+- Implements the following changes to URI authentication credentials validation, which
+  apply when creating a new ``mongoc_uri_t`` object from a connection string:
+
+  - Requires that usernames are non-empty for all authentication mechanisms.
+  - Validates username and password specification requirements and returns
+    a client error for the specified authentication mechanism.
+  - For the ``MONGODB-AWS`` authentication mechanism, the ``authSource`` value defaults to
+    ``$external``.
+  - Validates ``authMechanism`` values and returns a client error for invalid values.
+  - Validates ``authMechanismProperties`` values and returns a client error for invalid
```

Comment:
Suggest clarifying what "Correctly parses colon" means. Suggest removing mention of comma character. The comma character parsing behavior did not change. Documentation was only added to note it was disallowed.

```
- Validates ``authMechanismProperties`` values and returns a client error for invalid properties for the specified authentication mechanism
  - Now correctly parses colon (``:``) characters within ``authMechanismProperties`` values:
      - Old behavior: ``authMechanismProperties=A:B,C:D:E,F:G`` is parsed as ``{'A': 'B', 'C': 'D:E,F:G'}``.
      - New behavior: ``authMechanismProperties=A:B,C:D:E,F:G`` is parsed as ``{'A': 'B': 'C': 'D:E', 'F': 'G'}``.
```